bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java

```diff
@@ -479,7 +480,12 @@ protected String executeSingle(ResourceBundle language, String[] labelFragments,
             throws InterpretationException {
         ArrayList<Item> items = getMatchingItems(language, labelFragments, command.getClass());
         if (items.size() < 1) {
-            throw new InterpretationException(language.getString(NO_OBJECTS));
+            if ((command != null) && (getMatchingItems(language, labelFragments, null).size() >= 1)) {
```

Comment:
Above `command.getClass()` is called, so if command is null, you don't reach this line at all.